doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -282,15 +282,15 @@ sub-directory`_ and are run via the `teuthology framework`_.
 .. _`teuthology framework`: https://github.com/ceph/teuthology
 
 The Ceph community has access to the `Sepia lab
-<https://wiki.sepia.ceph.com/doku.php>`_ where :ref:`testing-integration-tests` can be
-run on physical hardware. Other developers may add tags like "needs-qa" to your
+<https://wiki.sepia.ceph.com/doku.php>`_ where `Integration Testing` _ can be
```

Comment:
doesn't link correctly https://ceph--37949.org.readthedocs.build/en/37949/dev/developer_guide/basic-workflow/#integration-tests-aka-ceph-qa-suite

I think you need:

```suggestion
<https://wiki.sepia.ceph.com/doku.php>`_ where `Integration Testing`_ can be
```